Description: Don't fail tests because we don't ship test data

This applies to users running them; our build/autopkgtest re-enable
this and use the data from the source tree

Author: Yaroslav Halchenko
Forwarded: no

--- a/setup.cfg
+++ b/setup.cfg
@@ -61,7 +61,7 @@ markers =
     high_memory: mark a test as a high-memory only
     clipboard: mark a pd.read_clipboard test
 doctest_optionflags = NORMALIZE_WHITESPACE IGNORE_EXCEPTION_DETAIL
-addopts = --strict-data-files
+# addopts = --strict-data-files
 xfail_strict = True
 filterwarnings =
     error:Sparse:FutureWarning
